test/scss/fixtures/bitstyles-overrides.css

```diff
@@ -1899,8 +1899,8 @@ table {
     justify-content: start;
   }
 }
-.bsu-line-height-min {
```

Comment:
👀 what was that?

Reply:
Which part? :D This was the old version of the utility class — it was hard coded to just `min` (which is the minimum line-height where fonts are guaranteed to show the diacritics like `ü` or `é` without cutting them off.

This is in the “overrides” fixture, where each module has a variable overridden before the CSS is output. One of those is the namespace `bs` which is prefixed to the rest of the classname. In this case it was being incorrectly concatenated, so there’s no minus: `.bsu-line-height-min` should be `.bs-u-line-height-min` (and it is now! 🎉)

Reply:
> … In this case it was being incorrectly concatenated

This part.